tests/AbstractSerializerTest.php

```diff
@@ -447,9 +447,6 @@ public function serializableCallableProvider()
                 ], [
                     'callable' => [$this, 'serializableCallableProvider'],
                     'expected' => 'Callable Sentry\Tests\AbstractSerializerTest::serializableCallableProvider []',
-                ], [
-                    'callable' => [Client::class, 'getConfig'],
```

Comment:
You should replace this kind of callback with an other (non deprecated, sure) one